.NET/Microsoft.Recognizers.Text/Matcher/TrieTree.cs

```diff
@@ -5,11 +5,10 @@ namespace Microsoft.Recognizers.Text.Matcher
 {
     public class TrieTree<T> : AbstractMatcher<T>
     {
-        protected readonly Node<T> root = new Node<T>();
```

Comment:
Let's extract these changes (hiding a former protected field) to another PR and revert those in this branch.